### PR TITLE
Overlapping graph with extreme values

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**
fix https://github.com/bancorprotocol/carbon-app/issues/987
fix https://github.com/bancorprotocol/carbon-app/issues/972

- Initialize min & max differently if market price is below 1wei
- Change graph ratio to support large difference between price range and between market price & price.
- Ensure price initialization in production mode

### CODE REVIEW

File: src/components/strategies/create/overlapping/CreateOverlappingStrategy.tsx [hunk 1]
```diff
@@ -121,10 +130,14 @@ export const CreateOverlappingStrategy: FC<OverlappingStrategyProps> = (
   useEffect(() => {
     if (!quote || !base || marketPrice <= 0) return;
     if (!order0.min && !order1.max) {
-      const _marketPrice = new SafeDecimal(marketPrice);
-      const min = _marketPrice.times(0.999).toFixed(quote.decimals);
-      const max = _marketPrice.times(1.001).toFixed(quote.decimals);
-      if (isValidRange(min, max)) setOverlappingParams(min, max);
+      const { min, max } = getInitialPrice(new SafeDecimal(marketPrice), quote);
```
Comment (pingustar) [inline src/components/strategies/create/overlapping/CreateOverlappingStrategy.tsx:133]: I would move `new SafeDecimal(marketPrice)` into `getInitialPrice` and accept `number | string` as marketPrice argument type.

File: src/components/strategies/create/overlapping/CreateOverlappingStrategy.tsx [hunk 2]
```diff
@@ -44,6 +44,15 @@ export type SetOverlappingParams = (
   max: string
 ) => Promise<StrategyPrices | undefined>;
 
+const getInitialPrice = (marketPrice: SafeDecimal, quote: Token) => {
```
Comment (pingustar) [inline src/components/strategies/create/overlapping/CreateOverlappingStrategy.tsx:47]: is this only for quote? Might be worth to rename `quote` to `tkn` and make it a bit more universal.

**Review by pingustar — APPROVED** [review]
Code LGTM. Left some nit comments

File: src/components/strategies/create/overlapping/CreateOverlappingStrategy.tsx (hunk 2 above)
Comment (GrandSchtroumpf) [reply src/components/strategies/create/overlapping/CreateOverlappingStrategy.tsx:47]: All price are display in quote in the app so it'll always be quote

**Comment by ashachaf** [comment]
- [ ] when dragging the handles in the chart, the input prices do not change to reflect
- [ ] at some point, when dragging the handles, the price reset to a high value for some reason
https://github.com/bancorprotocol/carbon-app/assets/40430285/0989ddde-d4a3-483c-8284-4642ff42d1df

**Pull request merged**